package.json

```diff
@@ -12,8 +12,9 @@
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\"",
```

Comment:
I removed the `--fix` Flag here as we are already having a `format`-Script

Reply:
The format script runs prettier, though, which (while also helpful) won't necessarily fix `eslint` errors AFAIK.

Reply:
I'd suggest a script for `lint:fix` or something similar. What do you think about that?

Reply:
100% agree.